DOCUMENTATION.md

```diff
@@ -99,6 +100,20 @@ Name | Type | Description | `flatpak override` equivalent
 Talks | Input | Allow the application to talk to session services. <br /> <br /> For example, adding `org.freedesktop.Notifications` will allow the application to send notifications. | `--talk-name=[NAME]`
 Owns | Input | Allow the application to own session services under the given name. | `--own-name=[NAME]`
 
+
+### Portals
+
+Portals, also know as dynamic permissions, are granted by users during runtime.
+
+Name | Type | Description | Portal
+--- | --- | --- | ---
+Background | Toggle | Allow the application to run in the background | `org.freedesktop.portal.Background`
+Notifications | Toggle | Allow the application to send notifications | `org.freedesktop.portal.Notification`
+Microphone | Toggle | Allow the the application to listen to your microphone | `org.freedesktop.portal.Device`
+Speakers | Toggle | Allow the application to play sounds to your speakers | `org.freedesktop.portal.Device`
+Camera | Toggle | Allow the application to record videos with your camera | `org.freedesktop.portal.Device`
+Location | Toggle | Allow the application to access your location | `org.freedesktop.portal.Location`
```

Comment:
```suggestion
Background | Toggle | Allow the application to run in the background. | `org.freedesktop.portal.Background`
Notifications | Toggle | Allow the application to send notifications. | `org.freedesktop.portal.Notification`
Microphone | Toggle | Allow the the application to listen to your microphone. | `org.freedesktop.portal.Device`
Speakers | Toggle | Allow the application to play sounds to your speakers. | `org.freedesktop.portal.Device`
Camera | Toggle | Allow the application to record videos with your camera. | `org.freedesktop.portal.Device`
Location | Toggle | Allow the application to access your location. | `org.freedesktop.portal.Location`
```

The rest of the descriptions have a period at the end. It might be nitpicky, but it's for consistency.